modules/network-observability-resource-recommendations.adoc

```diff
@@ -0,0 +1,20 @@
+//module included in the following assemblies:
+// network_observability/configuring_operator.adoc
+
+:_content-type: REFERENCE
+[id="network-observability-resource-recommendations_{context}"]
+= Resource management and performance recommendations
+
+The amount of resources required by Network Observability depends on the size of your cluster and your requirements for ingesting and storing observability data. There are settings to consider that can help you get an optimal setup and observability.
+
+The following settings can help you manage resources and performance from the outset:
+   
+    eBPF Sampling:: You can set the Sampling specification, `spec.agent.ebpf.sampling`, to manage resources. Lower sampling values might consume a large amount of computational, memory and storage resources. You can mitigate this by specifying a sampling ratio value. A value of 100 means 1 flow every 100 is sampled. A value of 0 or 1 means all flows are captured. The lower the value, the increase in returned flows and the accuracy of derived metrics. By default, eBPF sampling is set to a value of 50, so 1 flow every 50 is sampled. Note that more sampled flows also means more storage needed. It is recommend to start with default values and refine empirically, to determine which setting your cluster can manage.
```

Comment:
```suggestion
    eBPF Sampling:: You can set the Sampling specification, `spec.agent.ebpf.sampling`, to manage resources. Lower sampling values might consume a large amount of computational, memory, and storage resources. You can mitigate this by specifying a sampling ratio value. For example, setting a value of `100` for [Y] means that [Z] samples 1 flow every 100 flows. A value of `0` or `1` means all flows are captured. The lower the value, the increase in returned flows and the accuracy of derived metrics. By default, eBPF sampling is set to a value of `50`, so 1 flow every 50 is sampled. Note that more sampled flows also means more storage needed. Consider starting with the default value and refine empirically, so that you determine which setting your cluster can manage.
```

Reply:
I highlighted values set, I assume in some sort of configuration file.

Reply:
> It is recommend to
This is the suggested format for recommendations, as far as I understand

>so that you determine
How about _in order to_ ?

Reply:
On the suggestion, not sure what the "\[Y\]" and "\[Z\]" are referring to.  
  
The statement is not suggesting whether 50 is low or not.  I suggest to use the word "smaller" instead of "lower" because the smaller the sampling value, the more (higher) sampling it does!  It's an inverse relationship so avoid "lower" to avoid confusion.

Reply:
Thanks for weighing in on this @stleerh! Addressed.